README.md

```diff
@@ -114,8 +186,11 @@ val flightNumber by columnOf(10045.0, Double.NaN, 10065.0, Double.NaN, 10085.0)
 val recentDelays by columnOf("23,47", null, "24, 43, 87", "13", "67, 32")
 val airline by columnOf("KLM(!)", "{Air France} (12)", "(British Airways. )", "12. Air France", "'Swiss Air'")
 
-// create dataframe
+// create the dataframe
 val df = dataFrameOf(fromTo, flightNumber, recentDelays, airline)
+
+// print the dataframe
```

Comment:
Here "the" is correct :) since you're referring to something that's already there. In the previous line you created something new